ci-operator/step-registry/openshift/upgrade/gcp/ovn/openshift-upgrade-gcp-ovn-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: openshift-upgrade-gcp-ovn
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-loki
+    - chain: ipi-conf-gcp
+    - ref: ovn-conf
+    - chain: ipi-install
```

Comment:
The `openshift-upgrade-gcp` template does:

    pre:
    - ref: ipi-install-loki
    - chain: ipi-gcp-pre-stableinitial

which seems to expand to something slightly different than this (even ignoring the `ovn-conf`). Is this intentional?

Reply:
no, good catch, we should use `ipi-install-stableinitial`